bench/commands/utils.py

```diff
@@ -98,7 +98,7 @@ def set_redis_socketio_host(host):
 	set_redis_socketio_host(host)
 
 
-@click.command('set-default-site', help="Set default site for bench")
+@click.command('set-default-site [DEPRECATED]', help="Set default site for bench")
```

Comment:
How about we just remove it completely?

Alternatively, we can execute `bench use` if this is executed. Although I'm leaning towards complete elimination.

Reply:
Also, update frappe docs along with the in-repo docs too.

Reply:
> How about we just remove it completely?
> 
> 
> 
> Alternatively, we can execute `bench use` if this is executed. Although I'm leaning towards complete elimination. 

Ok.